misk/src/main/kotlin/misk/web/AdminTabModule.kt

```diff
@@ -0,0 +1,32 @@
+package misk.web
+
+import misk.inject.KAbstractModule
+import misk.web.actions.AdminTab
+
+class AdminTabModule : KAbstractModule() {
+  override fun configure() {
+    // only bind tabs here that do not consume any endpoint
```

Comment:
wanna promote this to a class-level user-facing comment? Like “Binds the admin UI framework. Individual tabs should be bound with their other code.”